experimenter/tests/integration/nimbus/utils/create_experiment_via_api.py

```diff
@@ -0,0 +1,60 @@
+import sys
```

Comment:
Don't we already have all this, the default data and the feature config maps, in conftest?

Reply:
Oh this shouldn't have been added, whoops!